### DISCUSSION TITLE
multi transaction Update: move Time Input to next line

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#6558] [Ideas]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/59400cb5-d490-4e6a-920a-3c323042c94c)

I have a lot of data where transactions only happen at certain times. I need this to update that data from 1.7.1 onwards.

Right now, we need to fix one transaction at a time. A lot of data...ioi

Please review whether it is possible to add a function that changes only the value after T in TRANSDATE.

Additional) can also be used to initialize a lot of data to 00:00:00!

**Comment by n-stein** [comment]
#6581
